src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -443,19 +433,6 @@ export const NavBar: React.FC<NavBarPropsType> = ({
                     size="small"
                   />
                 </div>
-                <OverlayTrigger
-                  trigger="click"
-                  placement="bottom-end"
-                  overlay={MenuPopover}
-                  rootClose={true}
-                >
-                  <ButtonNG
-                    className="NavBar__menu--link NavBar__menu-icon"
```

Comment:
Check if these classNames are used anywhere, if they are not please remove them.

Reply:
fixed in 4d552ca862565cef16d8f1e6f6d650d2be3aa48e